QtCollider/widgets/QcTreeWidget.h

```diff
@@ -90,13 +104,22 @@ class QcTreeWidget : public QTreeWidget
   Q_INVOKABLE int columnWidth( int column );
   Q_INVOKABLE void setColumnWidth( int column, int width );
 
-Q_SIGNALS:
+  Q_INVOKABLE void scrollToItem(QcTreeWidget::ItemPtr, int);
+  Q_INVOKABLE void expandItem(QcTreeWidget::ItemPtr);
+  Q_INVOKABLE void collapseItem(QcTreeWidget::ItemPtr);
+  Q_INVOKABLE void setCurrentItemRow(QcTreeWidget::ItemPtr item);
+  Q_INVOKABLE void setCurrentItem(QcTreeWidget::ItemPtr item, int column);
 
-  void action();
-  void itemPressedAction();
-  void currentItemChanged();
-  void expanded(QcTreeWidget::ItemPtr);
-  void collapsed(QcTreeWidget::ItemPtr);
+Q_SIGNALS:
+  void itemPressed(QcTreeWidget::ItemPtr item, int column);
+  void itemClicked(QcTreeWidget::ItemPtr item, int column);
+  void itemDoubleClicked(QcTreeWidget::ItemPtr item, int column);
+  void action(QcTreeWidget::ItemPtr item, int column); //    void itemActivated(QTreeWidgetItem *item, int column)
```

Comment:
what's the point of this comment?

Reply:
That signal is redirected from a different signal from the base class. Added clarification to comment.